Changelog

```diff
@@ -5,454 +5,304 @@
+- :program:`celery report` now reports kernel version along with other
+  platform details.
 
-- `GreenletExit` is not in `__all__` in greenlet.py which can not be imported by Python 3.6.
-
-  The import was adjusted to work on Python 3.6 as well.
-
-  Contributed by **Hsiaoming Yang**
-
-- Fixed a regression that occured during the development of Celery 4.2 which caused `celery report` to crash when Django is installed.
-
-  Contributed by **Josue Balandrano Coronel**
+  Contributed by **Omer Katz**
 
-- Matched the behavior of `GroupResult.as_tuple()` to that of `AsyncResult.as_tuple()`.
+- Fixed chords with chains which include sub chords in a group.
 
-  The group's parent is now serialized correctly.
+  TODO: Elaborate about this https://github.com/celery/celery/pull/5222
 
-  Contributed by **Josue Balandrano Coronel**
+  Contributed by **Maximilien Cuony**
 
-- Use Redis coercion mechanism for converting URI query parameters.
+- Complex canvases with error callbacks no longer raises an :class:`AttributeError`.
 
-  Contributed by **Justin Patrin**
+  TODO: Elaborate about this https://github.com/celery/celery/pull/5297
 
-- Fixed the representation of `GroupResult`.
+  Contributed by **Manuel Vázquez Acosta**
 
-  The dependency graph is now presented correctly.
+- Added proper error messages in cases where app cannot be loaded.
 
-  Contributed by **Josue Balandrano Coronel**
+  Previously, celery crashed with an exception.
 
-Documentation, CI, Installation and Tests fixes:
+  We now print a proper error message.
 
+  Contributed by **Omer Katz**
 
-    - **Sammie S. Taunton**
-    - **Dan Wilson**
-    - :github_user:`pachewise`
-    - **Sergi Almacellas Abellana**
-    - **Omer Katz**
-    - **Alex Zaitsev**
-    - **Leo Singer**
-    - **Rachel Johnson**
-    - **Jon Dufresne**
-    - **Samuel Dion-Girardeau**
-    - **Ryan Guest**
-    - **Huang Huang**
-    - **Geoffrey Bauduin**
-    - **Andrew Wong**
-    - **Mads Jensen**
-    - **Jackie Leng**
-    - **Harry Moreno**
-    - :github_user:`michael-k`
-    - **Nicolas Mota**
-    - **Armenak Baburyan**
-    - **Patrick Zhang**
-    - :github_user:`anentropic`
-    - :github_user:`jairojair`
-    - **Ben Welsh**
-    - **Michael Peake**
-    - **Fengyuan Chen**
-    - :github_user:`arpanshah29`
-    - **Xavier Hardy**
-    - **Shitikanth**
-    - **Igor Kasianov**
-    - **John Arnold**
-    - :github_user:`dmollerm`
-    - **Robert Knight**
-    - **Asif Saifuddin Auvi**
-    - **Eduardo Ramírez**
-    - **Kamil Breguła**
-    - **Juan Gutierrez**
+- Added the :setting:`task_default_priority` setting.
+
+  TODO: Elaborate about this https://github.com/celery/celery/pull/5309
+
+  Contributed by **:github_user:`madprogrammer`**
+
+- Bump minimum required version of Kombu to 4.3 and Billiard to 3.6.
+
+  Contributed by **Asif Saif Uddin**
+
+- Fix memory leak.
+
+  TODO: Elaborate about this.
+
+  Contributed by **George Psarakis**.
```

Comment:
@georgepsarakis Would you mind doing the write up for this entry?
I did not follow the entire conversation closely.

Reply:
Sure, I can help. The solution is described here https://github.com/celery/celery/pull/4839#issuecomment-399633253

I would suggest something close to the following:

```
Reintroduce weak references to bound methods for AsyncResult callback promises, after adding full weakref support for Python 2 in `vine <https://github.com/celery/vine/tree/v1.2.0>`_. More details can be found `here <https://github.com/celery/celery/pull/4839>_.
```

I think we should also add the PR author to the contributor list, since they discovered the root cause for the problem.